packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -133,7 +125,7 @@ const getInitialState = ({ preSelectedVmIds, setHomeVmIdsSelection }) => {
   setHomeVmIdsSelection([]) // Clear preselected vmIds
   return {
     _displayAdvancedSettings: undefined,
-     _proxyId: undefined,
```

Comment:
Do you know why this was like that?

Reply:
I don't know how i do it but it was my bad.